### ISSUE TITLE
Need to add sudo apt-get update?

### ISSUE THREAD

**Issue by caternuson** [adafruit/actions-ci-circuitpython-libs#3]
For workflows that are calling `install.sh`, some are getting 404s:
```
Err:1 http://azure.archive.ubuntu.com/ubuntu bionic-updates/main amd64 libudev-dev amd64 237-3ubuntu10.33
  404  Not Found [IP: 52.168.50.79 80]
```
([link to full actions run](https://github.com/adafruit/Adafruit_CircuitPython_LSM303DLH_Mag/runs/432327501))

Tried a restart a few times and still got the 404. Maybe add a:
```
sudo apt-get update
```
as first thing here?
https://github.com/adafruit/actions-ci-circuitpython-libs/blob/9db11f4c2ec8a425521346fd404d77abc9d1f7e9/install.sh#L15

**Comment by ladyada** [comment]
yep wanna try it? or i can

**Comment by caternuson** [comment]
I'll give it a go. Trivial PR. Curious about what it would take to actually test it first though.

**Comment by ladyada** [comment]
for stuff like this i just do the PR and see if the PR passes :)

**Comment by caternuson** [comment]
Should be fixed by #4
